AllReadyApp/Web-App/AllReady/Startup.cs

```diff
@@ -371,6 +370,23 @@ private IContainer CreateIoCContainer(IServiceCollection services)
                 await sampleData.CreateAdminUser();
             }
         }
+
+        private static void CheckForHangfireDatabaseAndCreateIfItDoesNotExist()
+        {
+            using (var sqlConnection = new SqlConnection("Server=(localdb)\\MSSQLLocalDB;Integrated Security=true;MultipleActiveResultsets=true;"))

```

Comment:
@mgmccarthy they are running allReady locally but SQL Server doesn't exist on their local machine so they have the database hosted remotely. My understanding is that this would typically be an Azure database